packages/actions/src/CreateTransaction/createTransaction.spec.ts

```
@@ -1,294 +1,145 @@
-				origin: createAddress(addr),
-			},
-			evmOutput: {
-				execResult: {
-					executionGasUsed: 21000n,
-					returnValue: new Uint8Array(),
-				},
-			},
+			client.on('newPendingTransaction', onNewPendingTransaction)
+			const txRes = await createTransaction(client)(options)
+			assert(!('errors' in txRes), 'txRes.errors should be undefined')
 		})
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)